APMrover2/Log.cpp

```diff
@@ -93,15 +93,13 @@ int8_t Rover::erase_logs(uint8_t argc, const Menu::arg *argv)
 
 int8_t Rover::select_logs(uint8_t argc, const Menu::arg *argv)
 {
-    uint16_t bits;
+    uint16_t bits = 0;
```

Comment:
Yes, this variable isn't initialized to 0. And as @khancyr says, it is set to 0 next, so better to do it here.